lib/compiler/index.js

```diff
@@ -93,10 +94,10 @@ const compiler = {
 
     switch (options.output) {
       case "parser":
-        return eval(ast.code);
+        return eval(ast.code.toString());
```

Comment:
Is this is going to break `ts-pegjs`?  I guess `.toString` is safe to add to a string, but I wonder if there's anything else that will break.

Reply:
Are you worried that `ts-pegjs` might have replaced the `code` with its object? Yes, it should be upgraded to correctly handle the new `sourceMap` option (or new `"output-with-map"` output variant)

Reply:
Let's get @pjmolina 's opinion early.  We can move several of the functions in generate-js into utils to make them easier to reuse, if that would help.